index.js

```diff
@@ -1,55 +1,56 @@
-const express = require("express");
+const express = require('express');
 
-const PORT = 8080;
+const PORT = 8000;
 const app = express();
 
-var product = require('./src/data/products.json');
-var category = require('./src/data/categories.json');
+// Import dataService.js file.
```

Comment:
For better comment, describe what is inside `dataService.js`.

Reply:
Done.